CONTRIBUTING.md

```diff
@@ -12,80 +12,69 @@ This project is a regular [Kubernetes Operator](https://coreos.com/operators/)
 
 ### Installing the Operator SDK command line tool
 
-Follow the installation guidelines from [Operator SDK GitHub page](https://github.com/operator-framework/operator-sdk) or run `make install-sdk`.
+Follow the installation guidelines from [Operator SDK GitHub page](https://github.com/operator-framework/operator-sdk)
```

Comment:
Isnt this part of [this pr](https://github.com/jaegertracing/jaeger-operator/pull/1762) already?

Reply:
Yes, that's why I closed this one.  ;-)

Reply:
ups now i see.